.github/workflows/go.yml

```diff
@@ -15,6 +15,9 @@ jobs:
           go-version: "^1.18.0"
       - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v3.1.0
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: v1.29
```

Comment:
This is wrong. It is both a very old version number, and not related to the version of golangci-lint :-(